sphinx_gallery/tests/test_gen_rst.py

```diff
@@ -50,7 +50,7 @@
 def test_split_code_and_text_blocks():
     """Test if a known example file gets properly split"""
 
-    blocks = sg.split_code_and_text_blocks('examples/just_code.py')
+    blocks,_ = sg.split_code_and_text_blocks('examples/just_code.py')
```

Comment:
PEP8 wants a space after the comma. You should probably configure your editor to have flake8 checks, the investment in time is worth it.